CMakeLists.txt

```diff
@@ -84,3 +97,9 @@ ExternalProject_Add(
   TEST_COMMAND   ""
   INSTALL_COMMAND ""
 )
+
+add_custom_target(create_tar ALL COMMAND
+  mkdir -p assets &&
+	tar -C ${CMAKE_BINARY_DIR} -cvz --exclude='include' --exclude='*.cmake' --exclude='Makefile' --exclude='CMake*' -f "assets/contracts-${GIT_TAG}.tar.gz" "contracts" --transform='s/contracts/contracts-${GIT_TAG}/g'
+	)
+add_dependencies(create_tar contracts_unit_tests)
```

Comment:
It will better if target doesn't build by default. Just add extra target which creates `tar` assets and run cmake with providing arguments to build this extra target.

Reply:
Changed. Check pls